javascript/695-max-area-of-island.js

```diff
@@ -2,32 +2,35 @@
  * @param {number[][]} grid
  * @return {number}
  */
- var maxAreaOfIsland = function(grid) {
```

Comment:
Git doesn't like two files that have a diff with same name, but different case styling
https://github.com/neetcode-gh/leetcode/blob/main/javascript/695-Max-Area-Of-Island.js
https://github.com/neetcode-gh/leetcode/blob/main/javascript/695-max-area-of-island.js